extension/src/popup/basics/PopupWrapper/styles.scss

```diff
@@ -1,6 +1,6 @@
 .PopupWrapper {
   max-height: calc(var(--popup--height) - var(--bottom-nav--height));
   max-width: var(--popup--width);
-  overflow-y: scroll;
+  overflow-y: overlay;
```

Comment:
❓ noob css-er question: was trying to see what `overlay` does and didn't see it on the [mozilla css docs](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-y), will all modern browsers support?

Reply:
Ah, good catch - I didn't mean to include this. This is only supported in Chrome. It makes the scrollbar sit on top of the content rather than appearing off to the side (as it usually does). `Overlay` is the first one
<img width="364" alt="Screen Shot 2022-03-01 at 4 43 00 PM" src="https://user-images.githubusercontent.com/6789586/156255404-d380de46-65a3-457f-bf0f-70b3668fec64.png">
<img width="362" alt="Screen Shot 2022-03-01 at 4 43 20 PM" src="https://user-images.githubusercontent.com/6789586/156255416-e35c9885-e0fc-485a-8c86-692787b57292.png">